components/SideBar.vue

```diff
@@ -1,12 +1,12 @@
 <template>
   <CSidebar
     :visible="visible"
-    :unfoldable="largeScreen"
-    :overlaid="largeScreen"
+    :unfoldable="screenIsLarge"
+    :overlaid="screenIsLarge"
     class="sidebar-fixed border-end"
     @hide="handleHide"
   >
-    <CSidebarNav role="navigation">
+    <CSidebarNav role="navigation" data-testid="sidebar">
```

Comment:
is data-testid neccesary? can it be tartged without this in tests? i am just against data-testid because its something users cant see and imaginary of sorts... and tests should mimic what users have access too

Reply:
It's a handle for playwright to grab onto. Playwright has a limited range of locators, but it does also have 'getByRole' that we could use here - except that we have two things with a navigation role so it's not unique. Also, when the navbar is narrow, it has no text that we could have done getByText on.

Reply:
It happens that this is vestigial anyway.